examples/geomstats_in_pymanopt.py

```diff
@@ -0,0 +1,113 @@
+"""This very simple example demonstrates how geometries from geomstats can be
+used in pymanopt to perform optimization on manifolds. It uses the Riemannian
+steepest descent solver.
+
+The example currently requires installing the pymanopt HEAD from git:
+
+    pip install git+ssh://git@github.com/pymanopt/pymanopt.git
+"""
+
+import functools
+import os
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"  # NOQA
+
+import numpy as np
+from numpy import linalg as la, random as rnd
+from pymanopt import Problem
+from pymanopt.manifolds.manifold import Manifold
+from pymanopt.solvers import SteepestDescent
+
+from geomstats.geometry.hypersphere import Hypersphere
+
+
+def squeeze_ndarray(f):
+    """Decorator to remove singleton dimensions from a numpy.ndarray returned
+    by the decorated function.
+    """
+    @functools.wraps(f)
+    def inner(*args, **kwargs):
+        return np.squeeze(f(*args, **kwargs))
+    return inner
+
+
+class GeomstatsSphere(Manifold):
+    """A simple adapter class which proxies calls by pymanopt's solvers to
+    manifolds to the underlying geomstats Hypersphere class.
+    """
+
+    def __init__(self, n):
+        self._sphere = Hypersphere(n-1)
+
+    def norm(self, x, g):
```

Comment:
`x, g` is pymanopt's API, right?

Reply:
Yes, though the naming convention fairly inconsistently mirrors the one used in manopt. I've been meaning to unify and break with the manopt conventions in a future release anyway. I'll change this to match closer what you do in geomstats.